### PR TITLE
Merge unified config and aad manifest migration log

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: Upgrade-to-support-unify-config-and-aad-manifest.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
 > Important Notes: If you collaborate on this project with your co-workers, please ensure your team members update the Teams Toolkit extension to the latest version after committing the changes with this upgrade.
 
 ## Why upgrade
 Teams Toolkit continues to improve your Teams application development experience. We are upgrading the Teams app project structure so that you can:
 1. Use a consistent schema to manage configuration settings for local and remote environments.
 1. Use a single Teams application manifest template across local and remote environments.
-1. Use AAD manifest template to customize your AAD app.
-1. Allow to add single sign-on (SSO) feature for Bot (hosting on Azure App Service), Static Launch Page, and Embed existing web App
+1. Use AAD manifest template to customize your AAD app (For project contains SSO feature).
+1. Allow to add SSO feature for Bot (hosting on Azure App Service), Static Launch Page, and Embed existing web App.
```
Comment (MuyangAmigo) [inline Upgrade-to-support-unify-config-and-aad-manifest.md:10]: Suggestion: change to this: `Allow to add SSO for these scenarios and capabilities: Notification bot (with restify server), Command bot, Bot, Tab and Embed existing web app.`

File: Upgrade-to-support-unify-config-and-aad-manifest.md [hunk 2]
```diff
@@ -30,4 +32,4 @@ Depending on how different the local and remote manifests are, there are two opt
 If anything went wrong after the upgrade process, you could restore your old project configuration files by:
 * Copy the .backup/.fx folder to your project root path.
 * Copy the .backup/templates folder to your project root path.
-* Delete `config.local.json`, `manifest.template.json` and `aad.template.json` if needed.
+* Delete `config.local.json`, `manifest.template.json` and `aad.template.json` (For project contains SSO feature) if needed.
```
Comment (MuyangAmigo) [inline Upgrade-to-support-unify-config-and-aad-manifest.md:35]: `For` -> `for`

File: Upgrade-to-support-unify-config-and-aad-manifest.md [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
 
 > Important Notes: If you collaborate on this project with your co-workers, please ensure your team members update the Teams Toolkit extension to the latest version after committing the changes with this upgrade.
 
 ## Why upgrade
 Teams Toolkit continues to improve your Teams application development experience. We are upgrading the Teams app project structure so that you can:
 1. Use a consistent schema to manage configuration settings for local and remote environments.
 1. Use a single Teams application manifest template across local and remote environments.
-1. Use AAD manifest template to customize your AAD app.
-1. Allow to add single sign-on (SSO) feature for Bot (hosting on Azure App Service), Static Launch Page, and Embed existing web App
+1. Use AAD manifest template to customize your AAD app (For project contains SSO feature).
```
Comment (MuyangAmigo) [inline Upgrade-to-support-unify-config-and-aad-manifest.md:9]: `For` -> `for`

File: Upgrade-to-support-unify-config-and-aad-manifest.md (hunk 1 above)
Comment (SLdragon) [reply Upgrade-to-support-unify-config-and-aad-manifest.md]: Thank you, updated

File: Upgrade-to-support-unify-config-and-aad-manifest.md (hunk 2 above)
Comment (SLdragon) [reply Upgrade-to-support-unify-config-and-aad-manifest.md]: Updated

File: Upgrade-to-support-unify-config-and-aad-manifest.md (hunk 3 above)
Comment (SLdragon) [reply Upgrade-to-support-unify-config-and-aad-manifest.md]: Updated

**Pull request merged**